lib/validation/params.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Litmus.Validation.Params do
+  @moduledoc false
+
+  @doc """
+  Module for checking if data consists of parameters that have not been defined in schema
+  """
+
+  @spec check_additional_params_error(map, map) :: tuple
```

Comment:
I think `additional` is misleading. What about `validate_allowed_params`?

Reply:
sure

Reply:
I'm thinking this function could just be a private function within the top level `Litmus module`.

Reply:
And the return type should be `:ok | {:error, String.t()}`. No need to return `{:ok, nil}` as a tuple.